### PR TITLE
used css media query and removed js in Header.jsx

### PR DESCRIPTION
**Pull request by kirti75waytech**
@MaksymTaratuta55 @thecodeclimber I have resolved issue #48 under this PR. Kindly review it and let me If any modifications are required.
Thanks

### CODE REVIEW

File: src/components/layouts/main/header/Header.jsx [hunk 1]
```diff
@@ -10,35 +9,13 @@ import {
...
-
-  const handleMobileTabletNavBar = () => {
-    if (navbarOpen && window.innerWidth < 1024) setNavbarOpen(false);
-  };
-
-  const resize = () => {
-    if (!navbarOpen && window.innerWidth >= 1024) setNavbarOpen(true);
-    handleMobileTabletNavBar();
-  };
+  const [navbarClose, setnavbarClose] = useState(true);
```
Comment (MaksymTaratuta55) [inline src/components/layouts/main/header/Header.jsx:15]: Please change setnavbarClose to "setNavbarClose"

File: src/components/layouts/main/header/Header.jsx [hunk 2]
```diff
@@ -10,35 +9,13 @@ import {
...
 
   const toggleNavbar = (open = false) => {
```
Comment (MaksymTaratuta55) [inline src/components/layouts/main/header/Header.jsx:17]: Please change "open" to "status". And use useCallback

File: src/components/layouts/main/header/HeaderNavBar.jsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ const NavBar = (props) => {
   ];
 
   const handleMenuDropDown = (id) => {
```
Comment (MaksymTaratuta55) [inline src/components/layouts/main/header/HeaderNavBar.jsx:26]: Please remove "id" if it doesn't use.
Please use useCallback

Comment (MaksymTaratuta55) [inline src/components/layouts/main/header/HeaderNavBar.jsx:26]: Please check above
```
const data = {};
  data.menu = [
    {
      name: "Home",
      id: SECTION_IDS.HOME,
    },
    {
      name: "How It Works",
      id: SECTION_IDS.HOW_IT_WORKS,
    },
    {
      name: "Our Founder",
      id: SECTION_IDS.OUR_FOUNDER,
    },
    {
      name: "Get Started",
      id: SECTION_IDS.CONTACT,
    },
  ];
```

**Review by MaksymTaratuta55 — CHANGES_REQUESTED** [review]
@kirti75waytech, please check my comments

File: src/components/layouts/main/header/Header.jsx (hunk 1 above)
Comment (kirti75waytech) [reply src/components/layouts/main/header/Header.jsx]: @MaksymTaratuta55 I have updated the above changes.

File: src/components/layouts/main/header/HeaderNavBar.jsx (hunk 3 above)
Comment (kirti75waytech) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @MaksymTaratuta55 I have deleted the id as it is not in use right now.

Comment (kirti75waytech) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @MaksymTaratuta55 Can you brief me on this issue. What changes do we required here?

File: src/components/layouts/main/header/Header.jsx (hunk 2 above)
Comment (kirti75waytech) [reply src/components/layouts/main/header/Header.jsx]: @MaksymTaratuta55 I have updated the above mentioned changes.

File: src/components/layouts/main/header/HeaderNavBar.jsx (hunk 3 above)
Comment (MaksymTaratuta55) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @kirti75waytech, I mean it shouldn't be in the component

Comment (MaksymTaratuta55) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @kirti75waytech, please use useCallback

File: src/components/layouts/main/header/Header.jsx (hunk 2 above)
Comment (MaksymTaratuta55) [reply src/components/layouts/main/header/Header.jsx]: @kirti75waytech, please use useCallback

File: src/components/layouts/main/header/HeaderNavBar.jsx (hunk 3 above)
Comment (kirti75waytech) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @MaksymTaratuta55 yes, I reported this issue earlier in issue: #39. we can solve this kind of improper merges here in this issue. Please let me know If I need to change it here or we can create another issue for this.

Comment (MaksymTaratuta55) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @kirti75waytech, Yes, fix it here please

Comment (kirti75waytech) [reply src/components/layouts/main/header/HeaderNavBar.jsx]: @MaksymTaratuta55 I have updated the above mentioned changes.

**Comment by kirti75waytech** [comment]
> @kirti75waytech, please check my comments

@MaksymTaratuta55 I have updated all the changes, you have suggested in this PR. Kindly review it and merge this PR.

**Pull request merged**